src/components/ChatLog.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { useState } from 'react';
+import './ChatLog.css';
+import ChatEntry from '../components/ChatEntry';
+
+const ChatLog = (props) => {
+    return props.entries.map((message) => {
+      return <ChatEntry incrementLikes={props.incrementLikes} decrementLikes={props.decrementLikes} sender={message.sender} body={message.body} timeStamp={message.timeStamp} key={message.id} />
+      
+    })
+};
+
```

Comment:
👀 We should add prop type for this component as well. We can make use of the `arrayOf` and `shape` `PropTypes` helpers to describe what we expect the array of message data to look like. And we can use `PropTypes.func` for any callback functions.